source/AI.cpp

```diff
@@ -714,6 +714,15 @@ void AI::Step(const PlayerInfo &player, Command &activeCommands)
 			continue;
 		}
 
+		if(isPresent && personality.IsSecretive() && !isStranded)
```

Comment:
This means a `secretive` ship wouldn't maintain that secrecy if it were in need of fuel?

Reply:
I inspired myself too much of these other personalities!